docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md

```diff
@@ -164,6 +164,11 @@ Before starting this procedure, set up your zones according to [Configure zone-a
 
 1. Upgrade the installation with the `helm` command using your regular command line flags.
 
+   This step also removes the Service and StatefulSet manifests of the old non zone-aware alertmanagers.
```

Comment:
@dimitarvdimitrov small suggestion. I am thinking why don't we add one distinct step to check if there is non zone-aware resources that are not removed instead of having this additional paragraph? The distinct step is more helpful to make users aware of an additional action that they need to check in this migration step.

As an example:

10. Check if there are non zone-aware store gateway resources that are not removed. In some cases, such as when using Helm from Tanka, these resources will not be automatically deleted from your Kubernetes cluster. when doing migration to zone-aware setup using helm. This can cause some of the pods to get scrapped multiple times when using the Prometheus operator for metamonitoring.

     - If non zone-aware store-gateways resources were not removed, you need to manually prune the old Service and StatefulSet. 
     - Wait for non zone-aware store-gateways resources to terminate

PS: need to grammar proof my suggestion if you consider this.

Reply:
yes, sounds good. please see the latest changes